src/components/DynamicCalendar/eventCal/DayView.js

```diff
@@ -35,13 +45,22 @@ export default class DayView extends React.PureComponent {
 
   componentWillReceiveProps(nextProps) {
```

Comment:
https://legacy.reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops

This lifecycle hook is deprecated and from what the documentation says, the non-prefixed version probably isn't being executed in React 17 and 18 (we're using React 18 for native and web rendering of our applications now, though significantly not in builder).

That said, perhaps our use of React Native and React Native Web means it's still running?  I'm honestly not sure.

Would it be worth adding the `UNSAFE_` prefix and re-testing?